### DISCUSSION TITLE
Übertragung von Punkten auf die nächste Seite

### DISCUSSION THREAD

**Discussion by Efendi5** [neuefische/questions#561] [Web] [answered]
Hallo,
gerade arbeite ich an meiner Result List, um die Punkte die gemacht wurden in meiner Quiz-App von der QuizPage auf die ResultlistPage zu übertragen. Mit dem NextButton pushe ich mithilfe vom router auf die ResultListPage und bekomme folgenden Fehler(siehe aufgeführte Links).
Der push auf die andere Seite funktioniert zwar und ich kann die Seite auch bearbeiten, aber es ploppt direkt der Fehler auf und die Punkte aus der QuizPage werden nicht übertragen.

Ich habe meinen aktuellen Stand committed und gepusht. Der Branch heißt ResultList
[Hier](https://github.com/Efendi5/capstone-project/pull/10)
hier auf nochmal der Vercel Link um den Fehler direkt in der Console zu sehen.
[Hier](https://capstone-project-three-sepia.vercel.app/)

TypeError: can't access property "question", _db_data__WEBPACK_IMPORTED_MODULE_8__.questions[index] is undefined

Source

components/QuizCardPage/QuizForm/index.js (19:19) @ questions

  17 | const [index, setIndex] = useState(0);
  18 | const [clickCounter, setClickCounter] = useState(0);
> 19 | const question = questions[index].question;
     |                 ^
  20 | 
  21 | return (
  22 |   <>

Vielen Dank für eure Hilfe!

**Comment by djfarly** [comment, marked as answer]
Hej @Efendi5,

it looks like your push is not going to the correct path. The QuizForm is rendered which expects to read the question with the index `index`. You array is out of bounds though and thus reading `question` from `questions[index]` throws an error.

Your results page is probably not supposed to render the quiz form at all. On vercel I am redirected to `/` instead of `/resultPage`.

**Reply by Efendi5** [reply]
Ich habe leider nicht verstanden was ich nun ändern muss, damit die Übertragung funktioniert. (Es liegt nicht am Englisch)

**Reply by djfarly** [reply]
Try to add a `?` to `const question = questions[index]?.question;` this way your code should crash - allowing you to investigate why you end up on the home page (`/`) again, instead of `/resultPage`.
